src/main/java/org/dinosaur/foodbowl/domain/comment/dto/CommentResponse.java

```diff
@@ -0,0 +1,25 @@
+package org.dinosaur.foodbowl.domain.comment.dto;
+
+import java.time.LocalDateTime;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.dinosaur.foodbowl.domain.comment.entity.Comment;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class CommentResponse {
+
+    private Long id;
+    private Long postId;
+    private String message;
+    private LocalDateTime createdAt;
+    private LocalDateTime updatedAt;
+
+    public static CommentResponse from(Comment comment) {
+        return new CommentResponse(comment.getId(), comment.getPost().getId(), comment.getMessage(),
+                comment.getCreatedAt(), comment.getUpdatedAt());
```

Comment:
```suggestion
        return new CommentResponse(
                comment.getId(),
                comment.getPost().getId(),
                comment.getMessage(),
                comment.getCreatedAt(),
                comment.getUpdatedAt()
        );
```
저는 가독성을 위해 매개변수로 인해 줄바꿈이 발생하는 경우 위와 같은 방법을 사용하는데 그레이 생각은 어떤가요?!

Reply:
좋습니다 ㅎㅎ